tests/cache/tests.py

```diff
@@ -1185,9 +1185,8 @@ def test_invalid_keys(self):
     def test_memcached_uses_highest_pickle_version(self):
         # Regression test for #19810
         for cache_key, cache_config in settings.CACHES.items():
-            if cache_config['BACKEND'] == 'django.core.cache.backends.memcached.MemcachedCache':
-                self.assertEqual(caches[cache_key]._cache.pickleProtocol,
-                                 pickle.HIGHEST_PROTOCOL)
+            self.assertEqual(caches[cache_key]._cache.pickleProtocol,
```

Comment:
Use a single line `self.assertEqual(caches[cache_key]._cache.pickleProtocol, pickle.HIGHEST_PROTOCOL)`. I see this is changed in a later commit but it seems the skipUnless isn't needed? Maybe we could just skip this commit since it's immediately moved anyway.